Makefile

```diff
@@ -0,0 +1,25 @@
+BIN=node_modules/.bin
```

Comment:
Hm, this was removed in #107, this PR is adding it back.

Could you [sync your fork](https://help.github.com/articles/syncing-a-fork/), and try to use https://github.com/acdlite/redux-actions/blob/master/package.json#L7 to build the UMD?